cfgov/v1/jinja2/v1/_includes/organisms/image-and-text-50-50.html

```diff
@@ -0,0 +1,59 @@
+{# ==========================================================================
+
+   image-and-text-50-50.render()
```

Comment:
Ah. Didn't realize that's what the rule was for macros. I'll double check to make sure that's consistent in the project.